packages/blade/src/_helpers/storybook/FigmaEmbed.tsx

```diff
@@ -14,12 +14,17 @@ function FigmaEmbed(props: { src: string; title: string }): JSX.Element {
       />
       <Description
         markdown={dedent`
-          > **Note**
-          > 
-          > Currently the designs are only accessible to Razorpay Employees.
-          >
-          > For Razorpay Employees,<br/>
-          > Figma Embed may not work with adblockers 😿. You can [View Design on Figma](${props.src}) or pause adblockers for this domain.
+        <small>
+          <details>
+            <summary style="cursor: pointer">Figma not loading?</summary>
+
+            > Currently the designs are only accessible to Razorpay Employees.
+            >
+            > For Razorpay Employees,<br/>
+            > Figma Embed may not work with adblockers 😿. You can [View Design on Figma](${props.src}) or pause adblockers for this domain.
```

Comment:
Unrelated Change: This poor task of collapsing the figma note was getting pushed in sprints lol. I was anyway touching the storybook parts here so did it. 

It collapses now so no need of unnecessary info

<img width="849" alt="image" src="https://user-images.githubusercontent.com/30949385/222671126-19633bca-885b-4d7d-852a-d392a22dbe70.png">

Reply:
TODO: Replace with Accordion later 😆

Reply:
Time to setup blade coverage on our documentation and make graphs go 📈